react/components/AppImagesSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FunctionComponent, useMemo } from 'react'
+import useProduct from 'vtex.product-context/useProduct'
+import { SliderLayout } from 'vtex.slider-layout'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = ['appImage'] as const
+
+const sortImages = (
+  images: Image[],
+  order: 'ASC' | 'DESC',
+  orderBy: keyof Image
+) => {
+  const sortedImages = images.sort((imageA, imageB) =>
+    imageA[orderBy].localeCompare(imageB[orderBy])
+  )
+  const numberOfImages = sortedImages.length
+  return order === 'ASC'
+    ? sortedImages
+    : sortedImages.map((_, index) => sortedImages[numberOfImages - index - 1])
+}
+
+interface Props {
+  filterPattern?: string
+  order?: 'ASC' | 'DESC'
+  orderBy?: keyof Image
+  sliderLayoutProps?: any
+}
+
+const AppImagesSlider: FunctionComponent<Props> = ({
+  filterPattern = '',
+  order = 'ASC',
+  orderBy = 'imageId',
```

Comment:
Use constant values when values are constants

```suggestion
  order = ASCENDING_ORDER,
   orderBy = IMAGE_ID_KEY
```

or something like that

Reply:
Ok, but I think `IMAGE_ID_KEY` is not necessary because the orderBy type is `keyof Image`. 
`keyof Image` = `"cachedId" | "imageId" | "imageLabel" | "imageTag" | "imageText" | "imageUrl"` . `orderBy` can be any of these options and `typescript` helps with the check.